homeassistant/components/demo/__init__.py

```diff
@@ -188,6 +188,17 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     return True
 
 
+async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Unload a config entry."""
+    await asyncio.gather(
+        *(
+            hass.config_entries.async_forward_entry_unload(config_entry, component)
```

Comment:
Can't we use async_unload_platforms here?

Reply:
Not sure how I missed this comment! Done now.